scanner/utils.lua

```diff
@@ -19,7 +19,7 @@ end
 
 function utils.boardExists()
 	if Board == nil or GetCurrentMission() == nil then
-		return false, "Enter a mission"
+		return false, "Enter a Mission with Chen as a Pilot"
```

Comment:
I don't really agree with changing the boardExists function that so many scans use, to request Chen as a pilot. I can understand the idea; you're going to need Chen at some point anyways. Still, each instruction is made short and accurate, which is why there are two different instructions for "mission" vs "real mission".

Reply:
That's fair, and that's the reason I put it in a separate commit. Should be easy to revert.